### DISCUSSION TITLE
The calibration button has disappeared.

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#1340] [Q&A] [answered]
Hi,

I did a clean install of Windows 11.

But now, the "calibration" button has disappeared from GHELPER.
How can I display the calibration button?

Thank you for your help.

![Capture d’écran 2023-09-21 085741](https://github.com/seerge/g-helper/assets/142881210/989a41d1-7b5f-4440-bc20-ede4ad5e9810)

**Comment by seerge** [comment, marked as answer]
@Kikimahe hello, i have removed it for devices that don't report any fan curve from bios :) (even when they support setting a custom one)

App also has automatic calibration, and if it will spot your fans going faster than default 5800RPM it will update it's scales :)

**Comment by Kikimahe** [comment]
Ahhhhhhhhhhhhhhhhh ! 

For 3 or 4 days I thought I was going crazy.
I installed and uninstalled the 3 versions of Ghelper several times (0.122, 0.123, 0.124) by deleting all the Ghelper files to find my calibration button! ;-DDD

Thank you for your reply.
 I am finally reassured ! ;-)
